app/utils/string_formatters.js

```diff
@@ -0,0 +1,9 @@
+var Price = require('format-price');
+
+String.prototype.capitalise = function() {
```

Comment:
Lodash already has an implementation for capitalise: https://lodash.com/docs#capitalize